doc/guides/timers-in-node.md

```diff
@@ -0,0 +1,175 @@
+---
+title: Timers in Node.js
+layout: docs.hbs
+---
+
+# Timers in Node.js and beyond
+
+The Timers module in Node.js contains functions that execute code after a set
+period of time. Timers do not need to be imported via `require()`, since
+all the methods are available globally to emulate the browser JavaScript API.
+To fully understand when timer functions will be executed, it's a good idea to
+read up on the the Node.js
+[Event Loop](https://nodesource.com/blog/understanding-the-nodejs-event-loop/).
```

Comment:
Should link to https://github.com/nodejs/node/blob/master/doc/topics/the-event-loop-timers-and-nexttick.md, which is an updated and revised version of that blog post.